### ISSUE TITLE
Vuoi lavorarci su?

### ISSUE THREAD

**Issue by Bavuett** [minusyt23/rubo-gli-ebook#1]
Figo. Non ho mai pensato all'utilità di portarsi dietro i file "puliti" degli eBook scolastici (principalmente perché non li uso) ma rompere il pipo alle piattaforme è oggettivamente divertente. Hai intenzione di lavorarci su per strutturarlo meglio e renderlo più accessibile?

**Comment by minusyt23** [comment]
Beh... sì
Mi piacerebbe lavorarci su perchè in realtà mi serve ancora, ho altri libri da estrarre
e mi piacerebbe anche trovare metodi di convertire quei libri in svg a pdf con testo effettivamente selezionabile e con metadata corretto, insomma automatizzare un po' tutto... ma dopo averne scaricato un paio mi sono accontentato e ho lasciato perdere il progetto. 

Magari oggi farò qualcosa ma sono abbastanza sicuro che nel giro di poche settimane tornerà nello stato di prima, vedrò di documentare i procedimenti

**Comment by Bavuett** [comment]
Te la butto lì. Credo che rendere il progetto un'estensione del browser sarebbe comodo. Hai esperienza con lo sviluppo di estensioni di Chrome?

**Comment by minusyt23** [comment]
No, però di sicuro mi renderebbe la vita più facile. Ora sono impegnatissimo e non posso lavorarci ma intanto dò un'occhiata a come creare estensioni
